### ISSUE TITLE
1.4.1 crashes without Trinkets

### ISSUE THREAD

**Issue by Poopooracoocoo** [kennethsible/elytrapitch#5]
### Description
The new Trinkets integration in your mod causes a crash when you try to fly with elytra when Trinkets is not present.

### Steps to reproduce
1. Launch game with Elytra Pitch and dependencies
2. Create a creative world
3. Equip elytra
4. Glide with elytra

### Log

[latest.log](https://github.com/user-attachments/files/17182448/latest.log)

<details><summary>Details</summary>

```
[10:46:14] [main/INFO]: Loading Minecraft 1.21.1 with Fabric Loader 0.16.5
[10:46:14] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
[10:46:15] [main/INFO]: Loading 56 mods:
	- cloth-config 15.0.140
	   \-- cloth-basic-math 0.6.1
	- elytrapitch 1.4.1
	- fabric-api 0.105.0+1.21.1
	   |-- fabric-api-base 0.4.42+6573ed8c19
	   |-- fabric-api-lookup-api-v1 1.6.68+b559734419
	   |-- fabric-biome-api-v1 13.0.29+5bd9f1bc19
	   |-- fabric-block-api-v1 1.0.22+0af3f5a719
	   |-- fabric-block-view-api-v2 1.0.10+6573ed8c19
	   |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a719
	   |-- fabric-client-tags-api-v1 1.1.15+6573ed8c19
	   |-- fabric-command-api-v1 1.2.49+f71b366f19
	   |-- fabric-command-api-v2 2.2.28+6ced4dd919
	   |-- fabric-commands-v0 0.2.66+df3654b319
	   |-- fabric-content-registries-v0 8.0.16+b559734419
	   |-- fabric-convention-tags-v1 2.0.23+7f945d5b19
	   |-- fabric-convention-tags-v2 2.8.0+9d22c5fd19
	   |-- fabric-crash-report-info-v1 0.2.29+0af3f5a719
	   |-- fabric-data-attachment-api-v1 1.1.28+1daea21519
	   |-- fabric-data-generation-api-v1 20.2.20+16c4ae2519
	   |-- fabric-dimensions-v1 4.0.0+6fc22b9919
	   |-- fabric-entity-events-v1 1.7.0+2122d82819
	   |-- fabric-events-interaction-v0 0.7.13+ba9dae0619
	   |-- fabric-game-rule-api-v1 1.0.53+6ced4dd919
	   |-- fabric-item-api-v1 11.1.0+d5debaed19
	   |-- fabric-item-group-api-v1 4.1.5+3fc0e55219
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a719
	   |-- fabric-keybindings-v0 0.2.45+df3654b319
	   |-- fabric-lifecycle-events-v1 2.3.12+6c1df36019
	   |-- fabric-loot-api-v2 3.0.14+3f89f5a519
	   |-- fabric-loot-api-v3 1.0.2+3f89f5a519
	   |-- fabric-message-api-v1 6.0.13+6573ed8c19
	   |-- fabric-model-loading-api-v1 2.0.0+fe474d6b19
	   |-- fabric-networking-api-v1 4.3.0+c7469b2119
	   |-- fabric-object-builder-api-v1 15.2.0+40875a9319
	   |-- fabric-particles-v1 4.0.2+6573ed8c19
	   |-- fabric-recipe-api-v1 5.0.13+6508971219
	   |-- fabric-registry-sync-v0 5.1.3+60c3209b19
	   |-- fabric-renderer-api-v1 3.4.0+c705a49c19
	   |-- fabric-renderer-indigo 1.7.0+c705a49c19
	   |-- fabric-renderer-registries-v1 3.2.68+df3654b319
	   |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2e19
	   |-- fabric-rendering-fluids-v1 3.1.6+1daea21519
	   |-- fabric-rendering-v0 1.1.71+df3654b319
	   |-- fabric-rendering-v1 5.0.5+df16efd019
	   |-- fabric-resource-conditions-api-v1 4.3.0+8dc279b119
	   |-- fabric-resource-loader-v0 1.3.0+5659912919
	   |-- fabric-screen-api-v1 2.0.25+8b68f1c719
	   |-- fabric-screen-handler-api-v1 1.3.87+b559734419
	   |-- fabric-sound-api-v1 1.0.23+6573ed8c19
	   |-- fabric-transfer-api-v1 5.3.0+d831205f19
	   \-- fabric-transitive-access-wideners-v1 6.1.0+1daea21519
	- fabricloader 0.16.5
	   \-- mixinextras 0.4.1
	- java 21
	- minecraft 1.21.1
[10:46:16] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/C:/Users/anon/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.15.3+mixin.0.8.7/sponge-mixin-0.15.3+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[10:46:20] [main/INFO]: Compatibility level set to JAVA_17
[10:46:20] [main/INFO]: Compatibility level set to JAVA_21
[10:46:23] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[10:46:25] [Datafixer Bootstrap/INFO]: 226 Datafixer optimizations took 952 milliseconds
[10:46:35] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[10:46:35] [Render thread/INFO]: Setting user: anon
[10:46:36] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[10:46:36] [Render thread/INFO]: Backend library: LWJGL version 3.3.3-snapshot
[10:46:39] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric, cloth-config, elytrapitch, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader
[10:46:39] [Worker-Main-7/INFO]: Found unifont_all_no_pua-15.1.05.hex, loading
[10:46:39] [Worker-Main-4/INFO]: Found unifont_jp_patch-15.1.05.hex, loading
[10:46:42] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[10:46:42] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[10:46:43] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Speakers (Realtek(R) Audio)
[10:46:43] [Render thread/INFO]: Sound engine started
[10:46:43] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[10:46:44] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[10:46:44] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[10:46:44] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[10:46:44] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[10:46:44] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/paintings.png-atlas
[10:46:44] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[10:46:44] [Render thread/INFO]: Created: 64x64x0 minecraft:textures/atlas/map_decorations.png-atlas
[10:46:44] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[10:46:44] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[10:47:40] [Render thread/INFO]: Loaded 1290 recipes
[10:47:41] [Render thread/INFO]: Loaded 1399 advancements
[10:48:46] [Render thread/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 1.804 ms
[10:48:46] [Server thread/INFO]: Starting integrated minecraft server version 1.21.1
[10:48:46] [Server thread/INFO]: Generating keypair
[10:48:51] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[10:48:51] [Render thread/INFO]: Preparing spawn area: 2%
[10:48:52] [Render thread/INFO]: Preparing spawn area: 2%
[10:48:52] [Render thread/INFO]: Preparing spawn area: 2%
[10:48:53] [Render thread/INFO]: Preparing spawn area: 2%
[10:48:53] [Render thread/INFO]: Preparing spawn area: 18%
[10:48:54] [Render thread/INFO]: Preparing spawn area: 18%
[10:48:54] [Render thread/INFO]: Preparing spawn area: 18%
[10:48:55] [Render thread/INFO]: Preparing spawn area: 51%
[10:48:55] [Render thread/INFO]: Preparing spawn area: 51%
[10:48:56] [Render thread/INFO]: Preparing spawn area: 51%
[10:48:56] [Render thread/INFO]: Time elapsed: 4835 ms
[10:48:56] [Server thread/INFO]: Changing view distance to 12, from 10
[10:48:56] [Server thread/INFO]: Changing simulation distance to 12, from 0
[10:48:57] [Server thread/INFO]: anon[local:E:08908f75] logged in with entity id 5 at (-7.5, 63.0, -3.5)
[10:48:57] [Server thread/INFO]: anon joined the game
[10:48:58] [Render thread/INFO]: Loaded 2 advancements
[10:48:59] [Server thread/INFO]: Saving and pausing game...
[10:48:59] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[10:48:59] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[10:48:59] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[10:49:19] [Server thread/INFO]: anon has reached the goal [Sky's the Limit]
[10:49:20] [Render thread/INFO]: [System] [CHAT] anon has reached the goal [Sky's the Limit]
[10:49:20] [Render thread/INFO]: Loaded 7 advancements
[10:49:22] [Server thread/INFO]: Saving and pausing game...
[10:49:22] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[10:49:22] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[10:49:22] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[10:49:35] [Render thread/ERROR]: Unreported exception thrown!
java.lang.NoClassDefFoundError: dev/emi/trinkets/api/TrinketsApi
	at fatalflare.elytrapitch.ElytraPitch.lambda$onInitialize$5(ElytraPitch.java:212) ~[elytrapitch-1.4.1.jar:?]
	at net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback.lambda$static$0(HudRenderCallback.java:28) ~[fabric-rendering-v1-5.0.5+df16efd019-42a709e4f30c7f7e.jar:?]
	at net.minecraft.class_329.handler$zkg000$fabric-rendering-v1$render(class_329.java:1538) ~[client-intermediary.jar:?]
	at net.minecraft.class_329.method_1753(class_329.java:220) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:896) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1285) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:882) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:256) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.5.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
Caused by: java.lang.ClassNotFoundException: dev.emi.trinkets.api.TrinketsApi
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.16.5.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	... 14 more
[10:49:36] [Server thread/INFO]: Stopping server
[10:49:36] [Server thread/INFO]: Saving players
[10:49:36] [Server thread/INFO]: Saving worlds
[10:49:37] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[10:49:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[10:49:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[10:49:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (New World): All chunks are saved
[10:49:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[10:49:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[10:49:45] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
``` 

</details> 

### Crash report

[crash-2024-09-30_10.49.45-client.txt](https://github.com/user-attachments/files/17182454/crash-2024-09-30_10.49.45-client.txt)

<details><summary>Details</summary>

```
---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 2024-09-30 10:49:45
Description: Unexpected error

java.lang.NoClassDefFoundError: dev/emi/trinkets/api/TrinketsApi
	at fatalflare.elytrapitch.ElytraPitch.lambda$onInitialize$5(ElytraPitch.java:212)
	at net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback.lambda$static$0(HudRenderCallback.java:28)
	at net.minecraft.class_329.handler$zkg000$fabric-rendering-v1$render(class_329.java:1538)
	at net.minecraft.class_329.method_1753(class_329.java:220)
	at net.minecraft.class_757.method_3192(class_757.java:896)
	at net.minecraft.class_310.method_1523(class_310.java:1285)
	at net.minecraft.class_310.method_1514(class_310.java:882)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.ClassNotFoundException: dev.emi.trinkets.api.TrinketsApi
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 14 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at fatalflare.elytrapitch.ElytraPitch.lambda$onInitialize$5(ElytraPitch.java:212)
	at net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback.lambda$static$0(HudRenderCallback.java:28)
	at net.minecraft.class_329.handler$zkg000$fabric-rendering-v1$render(class_329.java:1538)
	at net.minecraft.class_329.method_1753(class_329.java:220)

-- Uptime --
Details:
	JVM uptime: 212.659s
	Wall uptime: 180.747s
	High-res time: 178.813s
	Client ticks: 3226 ticks / 161.300s
Stacktrace:
	at net.minecraft.class_310.method_1587(class_310.java:2501)
	at net.minecraft.class_310.method_54580(class_310.java:949)
	at net.minecraft.class_310.method_1514(class_310.java:905)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- Affected level --
Details:
	All players: 1 total; [class_746['anon'/5, l='ClientLevel', x=3.48, y=111.43, z=-3.57]]
	Chunk stats: 961, 325
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,76,0), Section: (at 0,12,0 in 0,4,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 412 game time, 412 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
	Tracked entity count: 154

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, cloth-config, elytrapitch, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader

-- System Details --
Details:
	Minecraft Version: 1.21.1
	Minecraft Version ID: 1.21.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 462422016 bytes (441 MiB) / 1168113664 bytes (1114 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
	Identifier: Intel64 Family 6 Model 142 Stepping 12
	Microarchitecture: Whiskey Lake
	Frequency (GHz): 2.11
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Intel(R) UHD Graphics
	Graphics card #0 vendor: Intel Corporation
	Graphics card #0 VRAM (MiB): 1024.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.101.2128
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Virtual memory max (MiB): 15884.23
	Virtual memory used (MiB): 8687.45
	Swap memory total (MiB): 7942.12
	Swap memory used (MiB): 253.12
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 19769.70, total: 227966.00
	Space in storage for workdir (MiB): available: 19769.70, total: 227966.00
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx4096m
	Fabric Mods: 
		cloth-config: Cloth Config v15 15.0.140
			cloth-basic-math: cloth-basic-math 0.6.1
		elytrapitch: Elytra Pitch 1.4.1
		fabric-api: Fabric API 0.105.0+1.21.1
			fabric-api-base: Fabric API Base 0.4.42+6573ed8c19
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.68+b559734419
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.29+5bd9f1bc19
			fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a719
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8c19
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.52+0af3f5a719
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.15+6573ed8c19
			fabric-command-api-v1: Fabric Command API (v1) 1.2.49+f71b366f19
			fabric-command-api-v2: Fabric Command API (v2) 2.2.28+6ced4dd919
			fabric-commands-v0: Fabric Commands (v0) 0.2.66+df3654b319
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.16+b559734419
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.23+7f945d5b19
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.8.0+9d22c5fd19
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a719
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.28+1daea21519
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.20+16c4ae2519
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b9919
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.7.0+2122d82819
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.13+ba9dae0619
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.53+6ced4dd919
			fabric-item-api-v1: Fabric Item API (v1) 11.1.0+d5debaed19
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.5+3fc0e55219
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.47+0af3f5a719
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.45+df3654b319
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.12+6c1df36019
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.14+3f89f5a519
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.2+3f89f5a519
			fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8c19
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 2.0.0+fe474d6b19
			fabric-networking-api-v1: Fabric Networking API (v1) 4.3.0+c7469b2119
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.2.0+40875a9319
			fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8c19
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.13+6508971219
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.1.3+60c3209b19
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.4.0+c705a49c19
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.7.0+c705a49c19
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.68+df3654b319
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2e19
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+1daea21519
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.71+df3654b319
			fabric-rendering-v1: Fabric Rendering (v1) 5.0.5+df16efd019
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.3.0+8dc279b119
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.3.0+5659912919
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.25+8b68f1c719
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.87+b559734419
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.23+6573ed8c19
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.3.0+d831205f19
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.0+1daea21519
		fabricloader: Fabric Loader 0.16.5
			mixinextras: MixinExtras 0.4.1
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21.1
	Launched Version: 1.21.1
	Launcher name: PrismLauncher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Intel(R) UHD Graphics GL version 3.2.0 - Build 31.0.101.2128, Intel
	Window size: 854x480
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Render Distance: 12/12 chunks
	Resource Packs: vanilla, fabric, cloth-config, elytrapitch, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader
	Current Language: en_us
	Locale: en_AU
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 8x Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
	Server Running: true
	Player Count: 1 / 8; [class_3222['anon'/5, l='ServerLevel[New World]', x=3.48, y=111.43, z=-3.57]]
	Active Data Packs: vanilla, fabric, fabric-convention-tags-v2
	Available Data Packs: bundle, fabric, fabric-convention-tags-v2, trade_rebalance, vanilla
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: 8288872574982624248
``` 

</details>

**Comment by kennethsible** [comment]
In your crash log, I see `java.lang.NoClassDefFoundError`, so I should be able to resolve the issue by adding the following before importing anything from `dev.emi.trinkets.api`:
```
import net.fabricmc.loader.api.FabricLoader;

if (FabricLoader.getInstance().isModLoaded("trinkets")) ...
```
I thought I tested 1.4.1 without Trinkets installed, but I must've made a mistake. I am out of town for a week, but I'll be able to update the mod when I am back. For now, just downgrade the previous version without Trinkets.

**Comment by kennethsible** [comment]
I pushed a commit that fixes the crash without Trinkets (e83f27a). You'll see these changes in the next release (v1.4.2).
